MarkdownView/markdownview/src/main/AndroidManifest.xml

```diff
@@ -1,11 +1,4 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="us.feras.mdv">
-
-    <application
-        android:allowBackup="true"
-        android:label="@string/app_name"
-        android:supportsRtl="true">
```

Comment:
Thanks for the note, yeah I need to merge this PR this week and see if there's other items I could also add with this update.